README.md

```diff
@@ -76,6 +76,28 @@ app.use('/admin/queues', router)
 
 That's it! Now you can access the `/admin/queues` route and you will be able to monitor everything that is happening in your queues 😁
 
+### Hosting router on a sub path
+
+If you host your express service on a sub path (ie. https://<server_name>/<base_path>/) then you can add the following code to provide the configuration to the bull-board router. In this example the sub path will be `taurus`.
+
+```js
+const { router } = require('bull-board');
+
+// ... express server configuration
+
+let basePath = 'taurus';
+
+app.use(
+  '/admin/queuestat',
```

Comment:
Is it should be `/admin/queues`? 
I think that the example should be simplified,
Mount under `/my-base-path`, and set the `proxyUrl` to it.